HHS/OPRE-OPS
# feat: ✨ adds FY Select to CANs List

## What changed

- adds Fiscal Year Select for CANs List

## Issue

#347 #2878

## How to test

1. Goto CANs list
2. see list filtered by FY
3. select another FY from drop-down
4. see list update based on selected FY
5. 💰 profit

## Screenshots

https://github.com/user-attachments/assets/4880b3ec-e145-4e87-969e-75998d41cf80

## Definition of Done Checklist
